bin/clover/src/pipeline-steps/assetSpecificOverrides.ts

```diff
@@ -125,6 +126,31 @@ const overrides = new Map<string, OverrideFn>([
 
     variant.sockets.push(socket);
   }],
+  ["AWS::Route53::HostedZone", (spec: ExpandedPkgSpec) => {
+    const variant = spec.schemas[0].variants[0];
+
+    const prop = variant.resourceValue.entries.find((p: ExpandedPropSpec) =>
+      p.name === "NameServers"
+    )
+
+    if (!prop) return;
+    const socket = createOutputSocketFromProp(prop);
+
+    variant.sockets.push(socket);
+  }],
+  ["AWS::Route53::RecordSet", (spec: ExpandedPkgSpec) => {
+    const variant = spec.schemas[0].variants[0];
+
+    const prop = variant.domain.entries.find((p: ExpandedPropSpec) =>
+      p.name === "ResourceRecords"
+    );
+
+    if (!prop) return;
+    const socket = createInputSocketFromProp(prop);
+    setAnnotationOnSocket(socket, { tokens: ["NameServers"] });
+
+    variant.sockets.push(socket);
+  }],
```

Comment:
I just opened [this](https://github.com/systeminit/si/pull/5595) that covers off on this case.

![image](https://github.com/user-attachments/assets/3d85fc53-9fdb-4ff3-9b24-d02ea67f0c67)